config/routes.rb

```diff
@@ -3,15 +3,25 @@
 
   namespace :api, defaults: { format: 'json' } do
     namespace :v1 do
+      resources :users
+      
       resources :quizzes do
-        resources :questions, shallow: true
+        
+        resources :questions, shallow: true do
+          resources :answers, shallow: true do
+            # TODO: define a way to handle this path too
```

Comment:
Not really necessarily as it introduces unneeded complexity to your PlayerAnswers controller.

If you would really want to do it, instead of using `@player` in your `index`/`create` methods, you could use a `@parent` which you can set in the `before_action` filter to a `Player` or an `Answer`, depending on which key is present in the `params` object.